src/client.js

```diff
@@ -21,13 +22,24 @@ const viewBuildTypes = [
 const client = mozaik => {
 
     mozaik.loadApiConfig(config);
+    const caFilePath = config.get('jenkins.customCa');
+    let _certificate;
+
+    if (caFilePath.length > 0) {
+        _certificate = fs.readFileSync(caFilePath)
```

Comment:
@plouc What will happen right now the node app will not be able to start and fail in registering jenkins. Sample failure:

```
╰─○ node app.js
info: registered API "github"
info: registered API "travis"
info: registered API "weather"
fs.js:584
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^

Error: ENOENT: no such file or directory, open 'does_not_exist.crt'
```

Do you want a nicer way of handling this?